make.ps1

```diff
@@ -154,7 +154,7 @@ function Check-Scripts-Command
 {
 	if ((Get-Command "luac.exe" -ErrorAction SilentlyContinue) -ne $null)
 	{
-		echo "Testing Lua scripts..."
+		Write-Host "Testing Lua scripts..."  -ForegroundColor Blue
```

Comment:
Double blank space here and in several other places.

Reply:
Updated